src/components/MovieForm/MovieForm.js

```diff
@@ -0,0 +1,310 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// import GENRE_LIST from '../../App';
+import './MovieForm.scss';
+import Select from "react-dropdown-select";
+
+const EMPTY_FORM = {
+    movieTitle: '',
+    releaseDate: '2024-03-08',
+    movieUrl: '',
+    rating: 0,
+    relevantGenres: [],
+    duration: 0,
+    description: '',
+}
+
+const GENRE_LIST = [
+    {   value: 1,
+        label: 'All',
+    },
+    {
+        id: 2,
+        label: 'Documentary'
+    },
+    {
+        id: 3,
+        label: 'Comedy'
+    },
+    {
+        id: 4,
+        label: 'Thriller'
+    },
+    {
+        id: 5,
+        label: 'Crime',
+    },
+    {
+        id: 6,
+        label: 'Action',
+    },
+    {
+        id: 7,
+        label: 'Drama',
+    }
+];
+
+class MovieForm extends React.Component {
+    state = {
+        form: this.props.movieData ? {
+            movieTitle: this.props.movieData.movieTitle || '',
+            releaseDate: this.props.movieData.releaseDate || '2024-03-08',
+            movieUrl: this.props.movieData.movieUrl || '',
+            rating: this.props.movieData.rating || 0,
+            relevantGenres: this.props.movieData.relevantGenres.map((genre) => {
+                return GENRE_LIST.find((item) => item.value === genre.value);
+            }).filter(Boolean)|| [],
+            duration: this.props.movieData.duration || 0,
+            description: this.props.movieData.description || '',
+        } : EMPTY_FORM,
+        genreList: [
```

Comment:
The same array is used in Modal.stories.js. Create a data folder, put genreList there and import it here and in Modal.stories.js.